examples/hif2a/fit_to_multiple_rbfes.py

```diff
@@ -125,20 +122,32 @@ def _save_forcefield(filename, ff_params):
 
 
 if __name__ == "__main__":
+    parser = ArgumentParser(description="Fit Forcefield parameters to hif2a")
+    parser.add_argument("--num-gpus", default=NUM_GPUS, help="Number of GPUs to run against")
+    parser.add_argument("--hosts", nargs="*", default=None, help="Hosts running GRPC worker to use for compute")
+    parser.add_argument("--param-updates", default=1000, type=int, help="Number of updates for parameters")
+    parser.add_argument("--seed", default=2021, type=int, help="Seed for shuffling ordering of transformations")
+    parser.add_argument("--config", default="intermediate", choices=["intermediate", "production", "test"])
+    args = parser.parse_args()
     # which force field components we'll refit
-    forces_to_refit = [nonbonded.AM1CCCHandler, nonbonded.LennardJonesHandler]
+    forces_to_refit = [AM1CCCHandler, LennardJonesHandler]
 
     # how much computation to spend per refitting step
-    # configuration = testing_configuration  # a little
-    # configuration = production_configuration  # a lot
-    configuration = intermediate_configuration  # goldilocks
-
-    # how many parameter update steps
-    num_parameter_updates = 1000
-    # TODO: make this configurable also
-
-    # set up multi-GPU client
-    client = CUDAPoolClient(max_workers=configuration.num_gpus)
+    configuration = None
+    if args.config == "intermediate": # goldilocks
+        configuration = intermediate_configuration
+    elif args.config == "test":
+        configuration = testing_configuration # a little
+    else:
```

Comment:
I would probably keep the assertion below and change this to an elif for production.